diagnostics.yml

```diff
@@ -287,10 +287,12 @@ extends:
                 displayName: 'Publish Bundled Tools'
                 condition: succeeded()
               - output: pipelineArtifact
-                artifact: Logs_Packaging_Signing
+                artifact: Logs_Packaging_Signing_Attempt$(System.JobAttempt)
```

Comment:
Minor nit: I just added the JobAttempt # and not the "Attempt" to the rest of the artifact names.  It is what the arcade templates do too.

Reply:
https://github.com/dotnet/arcade/blob/main/eng/common/templates-official/job/job.yml#L22 and runtime too does this.